.github/workflows/mysql_performance.yml

```diff
@@ -38,17 +38,14 @@ jobs:
           echo "TEMP_AWS_SESSION_TOKEN=${creds[2]}" >> $GITHUB_ENV
       - name: 'Run performance tests (OpenJDK)'
         run: |
-          ./gradlew --no-parallel --no-daemon test-performance-aurora-mysql
+          ./gradlew --no-parallel --no-daemon test-all-environments -Ptest-no-docker=true -Ptest-no-iam=true -Ptest-no-hikari=true -Ptest-no-secrets-manager=true -Ptest-no-graalvm=true -Ptest-no-mariadb-driver=true -Ptest-no-mariadb-engine=true -Ptest-no-pg-engine=true -Ptest-no-pg-driver=true
         env:
-          AURORA_MYSQL_CLUSTER_IDENTIFIER: ${{ secrets.AURORA_MYSQL_CLUSTER_IDENTIFIER }}-perf-${{ github.run_id }}
-          AURORA_MYSQL_USERNAME: ${{ secrets.AURORA_MYSQL_USERNAME }}
```

Comment:
There're system environment variables DB_USERNAME and DB_PASSWORD. Values should be provided if a cluster already exists. In case of a new cluster (temporary) the values are optional.